lib/dynflow/web/console.rb

```diff
@@ -37,6 +37,16 @@ class Console < Sinatra::Base
         erb :worlds
       end
 
+      post('/worlds/execution_items') do
+        @worlds = world.coordinator.find_worlds
```

Comment:
We should limit this to active executors only (so `find_worlds(true)`) as no reason for reaching to the client worlds.